workos/events.py

```diff
@@ -58,17 +74,16 @@ def list_events(
             dict: Events response from WorkOS.
         """
 
-        if limit is None:
-            limit = RESPONSE_LIMIT
-            default_limit = True
-
-        params = {
+        params: EventsListFilters = {
             "events": events,
             "limit": limit,
             "after": after,
-            "organization_id": organization_id,
+            "organization_id": organization,
             "range_start": range_start,
             "range_end": range_end,
+            # TODO: This is a hack, and it's wrong. Events does not support before or order
+            "before": None,
```

Comment:
I'm a bit unclear - why do we need to define `before` here?

Reply:
Our current list params typing forces it. We need to refactor but I'm kicking that can down the road to a future PR.